src/fhirConfig.ts

```diff
@@ -27,7 +27,24 @@ export interface OAuthStrategy {
  * http://www.hl7.org/fhir/smart-app-launch/conformance/index.html#using-well-known
  */
 export interface SmartStrategy extends OAuthStrategy {
-    capabilities: string[];
+    // https://www.hl7.org/fhir/valueset-smart-capabilities.html
+    capabilities: (
+        | 'launch-ehr'
+        | 'launch-standalone'
+        | 'client-public'
+        | 'client-confidential-symmetric'
+        | 'sso-openid-connect'
+        | 'context-passthrough-banner'
+        | 'context-passthrough-style'
+        | 'context-ehr-patient'
+        | 'context-ehr-encounter'
+        | 'context-standalone-patient'
+        | 'context-standalone-encounter'
+        | 'permission-offline'
+        | 'permission-patient'
+        | 'permission-user'
+        | string
+    )[];
     scopesSupported?: string[];
```

Comment:
Nit picky but could help out customers adding comments for each of these variables http://hl7.org/fhir/smart-app-launch/conformance/index.html

Also having the comments as /** */ above each variable will allow them to be viewable from the IDE (using just // doesn't show the comment when using)

```suggestion
/**
* array of scopes a client may request
*/
    scopesSupported?: string[];
```